Packetery/Checkout/etc/db_schema.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="packetery_packet_draft" resource="default" engine="innodb" comment="Packetery Packet Draft Table">
+        <column xsi:type="int" name="id" nullable="false" identity="true" unsigned="true" comment="Entity ID"/>
+        <column xsi:type="int" name="order_id" nullable="false" comment="Order ID"/>
```

Comment:
přidal bych cizí klíč s onDelete="CASCADE", aby když se smaže záznam v tabulce s objednávkou, smazali se automaticky také záznamy v této tabulce. také bych nastavil int unsigned, aby to odpovídalo datovému typu